zabbix_metrics_watcher.py

```diff
@@ -139,7 +144,7 @@ def send_zabbix_notification(self, zabbix_host, osbs_master, concurrent_builds):
                 logger.warn('Exit code: %s' % e.returncode)
                 logger.warn('Output: %s' % e.output)
 
-        sleep(10)
+        sleep(1)
```

Comment:
We're running in a single thread, so making long pauses makes metrics incorrect when a high number of builds is submitted - as we spend 10 secs to reset settings to zero after a build is complete. This is unnecessary - Zabbix can handle 1 sec delay correctly